src/connector/src/parser/unified/debezium.rs

```diff
@@ -32,8 +32,80 @@ use crate::source::cdc::build_cdc_table_id;
 use crate::source::cdc::external::mysql::{
     mysql_type_to_rw_type, timestamp_val_to_timestamptz, type_name_to_mysql_type,
 };
+use crate::source::cdc::external::postgres::{pg_type_to_rw_type, type_name_to_pg_type};
 use crate::source::{ConnectorProperties, SourceColumnDesc};
 
+/// Checks if a given default value expression is a constant value.
+/// Uses a fast heuristic approach based on syntax structure.
+fn is_constant_default_value(default_value_expression: &str) -> bool {
+    if default_value_expression.trim().is_empty() {
+        return false;
+    }
+
+    let expr = default_value_expression.trim();
+
+    // 1. Quick check: contains parentheses = function call
+    if expr.contains('(') {
+        return false;
+    }
+
+    // 2. Check if it's a literal constant
+    if is_literal_constant(expr) {
+        return true;
+    }
+
+    // 3. Check if it contains operators
+    if regex::Regex::new(r"[+\-*/<>=!&|]").unwrap().is_match(expr) {
```

Comment:
Why bother implementing a problematic parsing again? For example `2 ^ 3` is not covered.